oslat-client

```diff
@@ -104,8 +104,13 @@ echo -e "${CMD_OUTPUT}"
 HK_CPUS=$(echo -e "${CMD_OUTPUT}" | grep "final cpus:" | awk '{ print $3 }')
 echo "HK_CPUS: ${HK_CPUS}"
 
-# get the first CPU from HK_CPUS to use as the oslat main thread
-cpu_main_thread=$(echo ${HK_CPUS} | awk -F, '{ print $1 }')
+# get the first CPU from WORKLOAD_CPUS to use as the oslat main thread
+cpu_main_thread=$(echo ${WORKLOAD_CPUS} | awk -F, '{ print $1 }')
+echo "cpu-main-thread: ${cpu_main_thread}"
+
+# LATENCY_CPUS are the WORKLOAD_CPUS less the cpu main thread
+LATENCY_CPUS=$(echo ${WORKLOAD_CPUS} | cut -d ',' -f2-)
+echo "LATENCY_CPUS: ${LATENCY_CPUS}"
```

Comment:
I think we need to decide if we want to make this mandatory or optional (ie. controlled by a parameter).

Reply:
@rafaelfolco @atheurer ??

Reply:
I'm not sure what we want to do, I was hoping for some input from @atheurer ...